### PR TITLE
Update documentation for exploring API with Postman

### PR DESCRIPTION
**Pull request by ajayns**
Addresses issue mentioned in #486 

Changes:
* Uses Postman native app instead of Chrome Extension
* Updated images using Plone localhost
* Added a sample response screenshot

### CODE REVIEW

File: docs/source/exploring.rst [hunk 1]
```diff
@@ -3,7 +3,11 @@
 Explore the API using Postman
 =============================
 
-To discover the API interactively, the Chrome-Extension Postman_ is a suitable solution.
+To discover the API interactively, the Postman_ is recommended.
```
Comment (tisto) [inline docs/source/exploring.rst]: "the Postman" -> "using Postman"

File: docs/source/exploring.rst [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+
+.. admonition:: Note
+
+    The Chrome-Extension version of Postman is deprecated and it is recommended to use the native apps available instead.
```
Comment (tisto) [inline docs/source/exploring.rst]: "native apps" -> "native app"

Comment (ajayns) [reply docs/source/exploring.rst]: I used plural referring to the different apps for each OS. Should I still change it? @tisto

File: docs/source/exploring.rst (hunk 1 above)
Comment (ajayns) [reply docs/source/exploring.rst]: Will do! @tisto

**Comment by ajayns** [comment]
I've gone ahead and made the changes anyway. Please review @tisto @kakshay21 @jaroel

**Comment by ableeb** [comment]
@titso @jensens @ajayns - @ajayns has signed the contributors agreement and accepted his GitHub invitation. I'm not sure why the PCA verifier is still showing failed status?

**Comment by gforcada** [comment]
@ableeb one needs to close and reopen the pull request so that the verifiers (at least the ones coming from mr.roboto) are checked again.

**Comment by ajayns** [comment]
@gforcada @ableeb Thanks! That seemed to have fixed it.

**Pull request merged**